tests/requirements/reqs-sanic-20.12.txt

```diff
@@ -0,0 +1,3 @@
+sanic==20.12.2
```

Comment:
Unfortunately, there seems to currently be a dependency conflict between `sanic` and `sanic-testing`. When running 

     pip install sanic==20.12.2 sanic-testing==0.3.0

pip bails out with 

    ERROR: Cannot install sanic-testing==0.3.0 and sanic==20.12.2 because these package versions have conflicting dependencies.
    
    The conflict is caused by:
        sanic 20.12.2 depends on httpx==0.15.4
        sanic-testing 0.3.0 depends on httpx==0.16.*

It gets even worse when running

    pip install sanic sanic-testing

pip goes down a rabbit hole of trying to find some combination of versions that resolve the conflict. I had to kill the installation process after an hour.

(all of this happens with recent pip versions that use the new resolver, and might be a symptom of https://github.com/pypa/pip/issues/9187)

This makes our CI fail. Any idea if this is a known issue in the Sanic community?

Reply:
`sanic-testing` should only be used from v21.3 forward. 

20.12 and older has the testing client built into it.

Reply:
I was kind of hoping we wouldn't run into pypa/pip#9187

Let me get rid of `sanic-testing` for now. Until the `v21.3` gets out, I will keep the `sanic` support fixed to `v20.12.2` and get rid of the `latest` behavior